screenshots/configs/core_screenshot_config.yaml

```diff
@@ -208,6 +215,10 @@ secondary:
     file: pdf
     message: downloading PDF for TN secondary
 
+  TX:
```

Comment:
you might need to pull your local copy of the repo so that the TX change gets integrated?